mreg/api/permissions.py

```diff
@@ -159,6 +160,9 @@ def _deny_superuser_only_names(data=None, name=None, view=None):
             if 'host' in data:
                 name = data['host'].name
     if '*' in name:
+        # members of the dns wildcard group can create wildcard records below subdomains (*.sub.example.com)
+        if request_in_settings_group(request, DNS_WILDCARD_GROUP) and name.count('.') >= 3 and not '_' in name:
+            return False
```

Comment:
Instead checking for underscore here, move the wildcard check after the under underscore in SRV test. Wildcards are already blocked in SRVs, so thats safe.

Reply:
Thanks for the tip. Implemented it in this new commit:
https://github.com/unioslo/mreg/pull/432/commits/e5b5827c30004cfa467ad40f9578307ea77387f6